src/common/templates/includes/macros.njk

```diff
@@ -15,6 +15,53 @@
 	{% endfor %}
 {% endmacro %}
 
+{% macro departure_warning_dialog(__) %}
+    <dialog id="confirmWarnedDepartureDialog" class="govuk-grid-column-two-thirds">
+      <h2 class="govuk-error-message ">
+        Warning
+      </h2>
+      <p class="twoHourWarningText">{{ __('field_departure_date_2_hours_prior') }}</p>
+      <p class="fortyEightHourWarningText">{{ __('field_departure_date_too_far_in_future') }}</p>
+
+      <p>
+        sGAR will allow you to submit the GAR, but it is not recommended.
```

Comment:
is this 'not recommended' text something Sonali has requested, if not, it is a little ambiguous what is meant by it, and perhaps contrary to what Sonali was saying. She said that she wanted them to submit the GAR, definitely when < 2hrs but also when > 48 hrs.

Reply:
<img width="1440" alt="Screenshot 2024-04-02 at 17 01 02" src="https://github.com/UKHomeOffice/egar-public-site-ui/assets/123387957/8ef4f759-916b-4f17-8177-2724fe7ade45">

Change has been made.